### PR TITLE
Alter testfile

### PR DESCRIPTION
**Pull request by tborealis**

### CODE REVIEW

File: testfile [hunk 1]
```diff
@@ -0,0 +1 @@
+Test
```
Comment (tborealis) [inline testfile]: This will never work like this!

**Review by tborealis — COMMENTED** [review]
Better now

File: testfile [hunk 2]
```diff
@@ -1 +1 @@
-Test
```
Comment (DaveLiddament) [inline testfile]: This is wrong

File: failing-test [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/php
 <?php
 
-exit(1);
+exit(0);
```
Comment (DaveLiddament) [inline failing-test:4]: I'd make this 1

File: testfile (hunk 1 above)
Comment (DaveLiddament) [inline testfile]: ANother review

**Review by DaveLiddament — CHANGES_REQUESTED** [review]

File: failing-test (hunk 3 above)
Comment (tborealis) [reply failing-test:4]: 1 makes this fail

**Review by DaveLiddament — DISMISSED** [review]

**Review by DaveLiddament — APPROVED** [review]

**Pull request merged**